website/sidebars.js

```diff
@@ -511,6 +511,7 @@ module.exports = {
         "reference/warehouse-profiles/dremio-profile",
         "reference/warehouse-profiles/clickhouse-profile",
         "reference/warehouse-profiles/materialize-profile",
+        "reference/warehouse-profiles/rockset-profile",
```

Comment:
```suggestion
        "reference/warehouse-profiles/rockset-profile"
```

I think the comma is why the preview is failing

Reply:
I just removed the comma in case -- that's strange though, because if you look at the materialize-profile above, it has the same comma. That's why I included it

Reply:
Ah that is odd